.circleci/config.yml

```diff
@@ -449,70 +445,46 @@ jobs:
           paths:
             - /home/circleci/go/pkg/mod
 
-  golint:
-    docker:
-      - image: cimg/go:1.17.1
-    environment:
-      TMPDIR: /mnt/ramdisk
-    working_directory: /home/circleci/go/src/github.com/influxdata/influxdb
+  lint:
+    executor: cross-builder
     steps:
       - checkout
-      - restore_cache:
-          keys:
-            - influxdb-gomod-sum-{{ checksum "go.sum" }}
-      - run: |
-          # this is not in a seperate bash script because it isn't meant to be run on local.
-          # it just checks to make sure that the same major/minor version of go is used in the mod file as on ci
-          # to prevent accidentally checking in a wrong go mod version.
-          gomodversiondiff=$( go mod edit -go=$( go version | sed -n 's/^.*go\([0-9]*.[0-9]*\).*$/\1/p') -print |diff - go.mod )
```

Comment:
We're dropping this check?

Reply:
I can put it back, it'll actually get more useful as more things are added to the cross-builder base (previously it wasn't so helpful since you could update the executor of the `lint` job and still leave the others out-of-date)